### ISSUE TITLE
question about baselines

### ISSUE THREAD

**Issue by zzwjames** [BUPT-GAMMA/GraphPAR#1]
Hi, thanks for your work and codes.

I have one question, your framework use labels to tune the classifier, but if we have labels, why don't you compare FairGNN in the main table? Is that reasonable to use labels to tune the framework when targeting self-supervised pre-trained models?

Thanks!

**Comment by zhongjian-zhang** [comment]
Thank you for your question, and I apologize for the delayed response.

1. We compared FairGNN in Table 2.
2. Our focus is on the scenarios of pre-training and fine-tuning. In this setting, the pre-training phase is generally unsupervised, while the downstream task fine-tuning is supervised.

I hope the above response addresses your concerns!
